src/software/backend/wifi_backend.cpp

```diff
@@ -31,6 +34,17 @@ WifiBackend::WifiBackend(std::shared_ptr<const NetworkConfig> network_config)
 void WifiBackend::onValueReceived(TbotsProto::PrimitiveSet primitives)
 {
     primitive_output->sendProto(primitives);
+
+    if (sensor_fusion_config->OverrideGameControllerDefendingSide()->value())
+    {
+        defending_side_output->sendProto(*createDefendingSide(
+            sensor_fusion_config->DefendingPositiveSide()->value() ? FieldSide::POS_X
+                                                                   : FieldSide::NEG_X));
+    }
+    else
+    {
+        defending_side_output->sendProto(*createDefendingSide(false));
```

Comment:
CI failing here, passing in bool instead of enum

Reply:
fixed